src/app.py

```diff
@@ -1,16 +1,24 @@
-''' Entry file for backend of application '''
```

Comment:
why did you delete the docstring? Remember to run pylint before a major pull request.